pkg/gql/query/blocks.go

```diff
@@ -199,7 +224,7 @@ func resolveReward(p graphql.ResolveParams) (interface{}, error) {
 			return nil, errors.New("context does not store database conn")
 		}
 
-		reward := uint64(0)
+		reward := emissionAmount(b.Height)
```

Comment:
If this is  the generator reward value, we should subtract the Dusk foundation reward as per `90% of the total value goes to the generator (rounded up)`